app/views/posts/_list_item.haml

```diff
@@ -1,5 +1,6 @@
 %tr
-  - klass = cycle('even', 'odd')
+  - klass = [cycle('even', 'odd')]
```

Comment:
It does seem to work as is:

![image](https://user-images.githubusercontent.com/577128/27839536-e97f566c-60e9-11e7-9da4-0655bce93c0e.png)